### PR TITLE
feat(config): Add link to new decentralization post

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -265,8 +265,8 @@ module.exports = {
         '/docs/security-model/optimism-security-model.md',
         '/docs/security-model/bounties.md',
         [
-          'https://medium.com/ethereum-optimism/our-pragmatic-path-to-decentralization-cb5805ca43c1',
-          'Decentralizing Optimism'
+          'htthttps://dev.optimism.io/decentralization-roadmap/',
```
Comment (Nicca42) [inline src/.vuepress/config.js:268]:
```suggestion
          'https://dev.optimism.io/decentralization-roadmap/',
```

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Comment by qbzzt** [comment]
Thank you and sorry - I really ought to avoid working while tired.

**Review by Nicca42 — APPROVED** [review]

**Comment by zainbacchus** [comment]
Reopened here: https://github.com/ethereum-optimism/community-hub/pull/696

**Comment by Nicca42** [comment]
Closing for #696

**Pull request closed without merging**